src/components/ec-file-dropzone/ec-file-dropzone.vue

```diff
@@ -119,14 +132,36 @@ export default {
 
   &__img {
     @apply tw-fill-key-4 tw-mb-8;
+
+    &--is-disabled {
+      @apply tw-fill-gray-6;
+    }
+  }
+
+  &__title {
+    @apply tw-h3;
```

Comment:
Out of curiosity, why did you have to change the HTLM tag from `h3` to `div` and then apply the `tw-h3` class instead of leave it as a `h3`?

Reply:
We were discussing it with Milos and Derrick. Can look like an h3 but semantically this is not an h3 so we changed to plain div.

Reply:
Oh, I see. It's interesting. Until now I had only seen it from the perspective of the visual aspect, and it makes sense to look at it from that other point of view. Thanks for the explanation!